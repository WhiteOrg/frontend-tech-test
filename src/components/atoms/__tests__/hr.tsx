import React from "react";
import renderer from "react-test-renderer";
import Hr from "../hr";

describe("ATOMS: Render testing", () => {
  test("Should render Hr", () => {
    const component = renderer.create(
      <div>
        <Hr />
      </div>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
