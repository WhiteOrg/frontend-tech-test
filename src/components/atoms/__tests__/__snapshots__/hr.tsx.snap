// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATOMS: Render testing Should render Hr 1`] = `
<div>
  <hr
    className="sc-AxjAm himXKj"
  />
</div>
`;
